vscode/src/projectSystem.ts

```diff
@@ -132,14 +133,16 @@ async function readFileUri(
         uri: uri,
       };
     });
-  } catch (_err) {
+  } catch (err) {
     // `readFile` should throw the below if the file is not found
     if (
-      !(_err instanceof vscode.FileSystemError && _err.code === "FileNotFound")
+      !(err instanceof vscode.FileSystemError && err.code === "FileNotFound")
     ) {
-      log.error("Unexpected error trying to read file", _err);
+      log.error("Unexpected error trying to read file", err);
+      return null;
+    } else {
+      throw err;
```

Comment:
Maybe it's ok to always throw here? Makes it less likely errors will be missed and require looking at the logs.